requirements/base.in

```diff
@@ -1,12 +1,46 @@
-# Core requirements for using this application that are not provided by edx-platform
+# Core requirements for using this application.
 
-django-object-actions==0.10.0           # Object actions in Django admin
-djangorestframework-xml==1.3.0
-django-multi-email-field==0.5.1         # Multi email input field in Django admin
-jsondiff==1.1.1                         # Used to detect content metadata changes
-django-countries==4.6.1
+-c edx-platform-constraints.txt
+
+django-object-actions               # Object actions in Django admin
+git+https://github.com/edx/django-rest-framework-oauth.git@0a43e8525f1e3048efe4bc70c03de308a277197c#egg=djangorestframework-oauth==1.1.1
+djangorestframework-xml
+django-multi-email-field            # Multi email input field in Django admin
+jsondiff                            # Used to detect content metadata changes
+django-countries
 -e git+https://github.com/edx/TinCanPython.git@0.0.5.py.35#egg=tincan==0.0.5.py.35
-django-simple-history==2.7.0
-edx-rbac==0.2.1
+django-simple-history
+edx-rbac
```

Comment:
@brittneyexline is this safe to unpin? i've lost track of that conversation

Reply:
(and `edx-drf-extensions`)?

Reply:
we cannot unpin edx-rbac until ENT-2002 is addressed, so we should keep this at `0.2.1`